ui/StatusQ/src/StatusQ/Core/StatusProfileImageSettings.qml

```diff
@@ -19,7 +19,7 @@ QtObject {
         width: root.width
         height: root.height
         name: root.name
-        isImage: name.includes("data")
+        isImage: name.includes("data:image")
```

Comment:
Absolutely, just wasn't sure if there are cases where rendering an image could break because the condition wouldn't pass.
Will update. Thanks for taking the time and being resistant here.